tests/unit/Makefile.am

```diff
@@ -1,3 +1,6 @@
+EXTRA_DIST += tests/unit/CMakeLists.txt	\
+	tests/unit/test_thread_wakeup.c
```

Comment:
With https://github.com/balabit/syslog-ng/pull/2130, this workaround is not needed anymore. I will force push it out soon.